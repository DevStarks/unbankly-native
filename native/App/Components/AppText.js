import React, { Component } from 'react'
import { Text } from 'react-native'
import styles from './Styles/AppTextStyles'
import ExamplesRegistry from '../Services/ExamplesRegistry'

// Ignore in coverage report
/* istanbul ignore next */
ExamplesRegistry.addComponentExample('AppText', () =>
  <AppText>This is text</AppText>
)

class AppText extends Component {
  render () {
    return (
      <Text {...this.props} style={[styles.text, this.props.style]}>
        {this.props.children}
      </Text>
    )
  }
}

export default AppText
