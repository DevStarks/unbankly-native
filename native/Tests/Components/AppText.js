import 'react-native'
import React from 'react'
import AppText from '../../App/Components/AppText'
import renderer from 'react-test-renderer'

test('AppText component renders correctly', () => {
  const tree = renderer.create(<AppText>This is text</AppText>).toJSON()
  expect(tree).toMatchSnapshot()
})
