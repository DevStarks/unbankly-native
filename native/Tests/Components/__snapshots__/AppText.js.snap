// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppText component renders correctly 1`] = `
<Text
  style={
    Array [
      Object {
        "color": "#637381",
        "fontFamily": "Oswald-Light",
        "fontSize": 20,
      },
      undefined,
    ]
  }
>
  This is text
</Text>
`;
